docs/newbs_flashing.md

```diff
@@ -1,18 +1,39 @@
-# Flashing Your Keyboard 
+# Flashing Your Keyboard
 
-Now that you've built a custom firmware file you'll want to flash your keyboard. 
+Now that you've built a custom firmware file you'll want to flash your keyboard.
+
+### Put Your Keyboard into DFU (Bootloader) Mode
+
+In order to flash your custom firmware you must first put your keyboard into a special flashing mode. While it is in this mode you will not be able to type or otherwise use your keyboard. It is very important that you do not unplug the keyboard or otherwise interrupt the flashing process while the firmware is being written.
+
+Different keyboards have different ways to enter this special mode. If your PCB currently runs QMK, TMK, or PS2AVRGB (Bootmapper Client) and you have not been given specific instructions, try the following, in order:
+
+* Hold down both shift keys and press `Pause`
+* Hold down both shift keys and press `B`
+* Unplug your keyboard, hold down the Spacebar and `B` at the same time, plug in your keyboard and wait a second before releasing the keys
+* Unplug your keyboard, hold down the top or bottom left key (usually Escape or Left Control) and plug in your keyboard
+* Press the physical `RESET` button on the bottom of the PCB
+* Locate header pins on the PCB labeled `BOOT0` or `RESET`, short those together or to ground while plugging your PCB in
```

Comment:
`BOOT0` shouldn't go to ground unless you actually want it to boot into the normal user-mode code.

```suggestion
* Locate header pins on the PCB labeled `RESET` and `GND` (or `GROUND`) and short those together while plugging your PCB in
 
If you've attempted all of the above, and the main chip on the board says `STM32` on it, this may be a bit more complicated. Generally board designers will provide a `RESET` button, or potentially two pads on the PCB that can be temporarily connected, as outlined above. `STM32` devices may have different entry sequences into bootloader mode, so if you're certain that no reset functionality is present on the board, your best bet is to ask on Discord for assistance. When asking for assistance, it's likely some photos of the board will be asked for -- if you can get them ready beforehand it'll help move things along!
```

Reply:
I think this is a tad too wordy, and we should try to keep the info dump in newbs docs to a minimum. Maybe remove the middle bit about STM32 and board designers, but keep the Discord signpost?

Reply:
Finy be me.